app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+# Like model
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    Post.find_by(id: post_id).increment(:likesCounter).save
+  end
```

Comment:
- [ ] Optional

- **Just like we have in user and comment:**  `after_save :update_likes_counter`